Docs/user_guide/modules/terminologies.md

```diff
@@ -0,0 +1,50 @@
+# Terminologies
+
+## Overview
+
+The Terminologies module is a place where the loaded terminology and anatomical contexts can be explored and managed. The same widget (called terminology navigator) appears when editing the "color" of certain types of data nodes (such as models and markups) in the [Data](data.html) module as well as the structures in [Segment Editor](segmenteditor.html), and is used to set the identity and color of the nodes or segments in question.
+Default terminology contexts are the DICOM master list, which is a full list of terminology categories and types, and the 3D Slicer General Anatomy list, which is a subset of the master list, and the entries of which are associated with the labels in the GenericAnatomyColors color table (and the colors matched). Some terminology categories allow the selection of an anatomic region, which come from a different list. There is only one default anatomic context dictionary, which is also called the DICOM master list (but for anatomic contexts). Custom contexts can also be defined.
```

Comment:
Tried to rework some of the sentences here since it was a bit of a bear to read and understand.

```suggestion
Default terminology contexts are the DICOM master list, which is a full list of terminology categories and types, and the 3D Slicer General Anatomy list, which is a subset of the master list. The terminology entries are associated with the labels in the GenericAnatomyColors color table (and the colors matched). Some terminology categories allow the selection of an anatomical region, which come from a different list. There is only one default anatomical context dictionary, which is also called the DICOM master list. Custom contexts can also be defined.
```

Is there one or two? I was getting confused by how these two sentences relate.

> Default terminology contexts are the DICOM master list, which is a full list of terminology categories and types, and the 3D Slicer General Anatomy list, which is a subset of the master list.

> There is only one default anatomical context dictionary, which is also called the DICOM master list.